src/core/api/index.js

```diff
@@ -187,10 +190,12 @@ export function callApi({
           .then((jsonResponse) => ({ response, jsonResponse }));
       }
 
-      log.warn(
-        oneLine`Response from API was not JSON (was Content-Type:
-        ${contentType})`,
-        response,
```

Comment:
@kumar303 unfortunately, it is not even the length of a full URL sometimes (we have very long URLs). I looked at the JSON and it is not super useful because lots of data are internal HTTP stuff, and only a fraction of all the information could be relevant to us.

Reply:
Yes, I know, but I think the first 100 characters of the response could be useful in some cases. For example, knowing that it was JSON at all is a big clue. Seeing something like a fatal text error would be another clue. Seeing some generic HTML headers would give us yet more clues. Not seeing the response body at all would be unhelpful.

Reply:
Whoops, you're right. I misread you and mixed "body" and "object".